index.html

```diff
@@ -4975,6 +4975,637 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The <code>AudioWorklet</code> object allows importing user-supplied
+          Javascript code that runs on the audio <a>rendering thread</a>. This
+          processing mechanism ensures the synchronous execution of the script
+          code with other built-in <code>AudioNode</code>s in the audio graph.
+        </p>
+        <p>
+          The <code>AudioWorklet</code> object stores the definitions of a
+          custom audio processor and an <code>AudioWorkletNode</code> object
+          can be constructed based on the processor definition. Note that the
+          registered definition in an AudioWorklet is valid for the entire
+          global scope (i.e. <code>window</code>), therefore any
+          <a><code>AudioContext</code></a> instance in the same global scope
+          can access the registered definition to create a custom node.
+        </p>
+        <p>
+          After importing a module script into the
+          <a><code>AudioWorkletGlobalScope</code></a>, an instance of
+          <a><code>AudioWorkletNode</code></a> can be created in the main
+          global scope. At the same time, the user agent automatically creates
+          an instance of <a><code>AudioWorkletProcessor</code></a>. This
+          counterpart represents an audio processing module that runs on the
+          audio <a>rendering thread</a>. These two components can communicate
+          via message passing.
+        </p>
+        <dl title="interface AudioWorklet : Worklet" class="idl">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script and returns a promise when fully loaded.
```

Comment:
I don't think you want "when fully loaded".  I also think you need to describe what happens to cause the promise to be resolved or rejected. I assume it's resolved when the script is loaded successfully. and it's rejected if there are syntax errors or it doesn't exist or maybe some other conditions?